### PR TITLE
Project display + misc. changes

### PR DESCRIPTION
**Pull request by BrendanReilly**

### CODE REVIEW

File: src/XTMF2.Web.Components/Util/InputRequestDialog.razor.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        /// <summary>
+        /// Callback when a valid input submission was made.
+        /// </summary>
+        [Parameter]
+        public EventCallback<string> OnSubmit { get; set; }
+
+        protected InputModel InputModel = new InputModel();
+
+        private BSModal InputModal;
```
Comment (JamesVaughan) [inline src/XTMF2.Web.Components/Util/InputRequestDialog.razor.cs]: This variable name violates the coding standard for private variables.

File: src/XTMF2.Web/Pages/ProjectDisplay/ProjectDisplay.razor.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+using XTMF2.Web.Components.Util;
+
+namespace XTMF2.Web.Pages
+{
+    /// <summary>
+    ///     Single project view (page).
+    /// </summary>
+    public partial class ProjectDisplay
+    {
+        private InputRequestDialog InputDialog;
```
Comment (JamesVaughan) [inline src/XTMF2.Web/Pages/ProjectDisplay/ProjectDisplay.razor.cs]: This variable name violates the coding standard for private variables.

**Review by JamesVaughan — CHANGES_REQUESTED** [review]
Just 2 small style fixes and this should be ready to merge.

**Comment by BrendanReilly** [comment]
@JamesVaughan Pushed changes addressing style problems.

**Review by JamesVaughan — APPROVED** [review]

**Pull request merged**